src/test/kotlin/com/karumi/maxibonkata/Generators.kt

```diff
@@ -17,4 +17,17 @@ object Generators {
         override fun generate(): Developer =
                 Developer(Gen.string().generate(), Gen.choose(0, 7).generate())
     }
+
+    class DeveloperForMaxGenerator(val maxMaxibonsToGrab: Int) : Gen<Developer> {
+        override fun generate(): Developer =
+                Developer(Gen.string().generate(), Gen.choose(0, maxMaxibonsToGrab + 1).generate())
+    }
+
+    class ListOf<T>(val min: Int, val max: Int, val gen: Gen<T>) : Gen<List<T>> {
+        override fun generate(): List<T> {
+            val numberOfElements = Gen.choose(min, max + 1).generate()
```

Comment:
I had another two alternatives here:
```
Gen.list(gen).filter { it.size <= max && it.size >= min}.generate()
```
But we should minimice the filters to generators at the test could fail because it can not generate a very restrictive value.

```
Gen.listOf(gen).generate().subList(0, minOf(Gen.choose(min, max + 1).generate(), list.size))
```
The logic to create the sublists is error prone and we could generate invalid sizes of lists or OutOfBoundException if we do it wrong.

So I finally went for this approach.
Do you think that adding a comment in the code about the filters in the Generators could worth it?

Reply:
it's ok as is 😃